slick.remotemodel-yahoo.ts

```diff
@@ -0,0 +1,215 @@
+(function ($) {
+  /***
+   * A sample AJAX data store implementation.
+   * Right now, it's hooked up to load Hackernews stories, but can
+   * easily be extended to support any JSONP-compatible backend that accepts paging parameters.
+   */
+  function RemoteModel() {
+    // private
+    var PAGESIZE = 10;
+    var data: { [key: number]: any; length: number } = { length: 0 };
+    var h_request: NodeJS.Timeout | null = null;
```

Comment:
setTimeout in the dom returns a number, not NodeJS.Timeout

Reply:
I merged, can be fixed in another PR

Reply:
resolved by #14

Reply:
Hmmm, tried number but the return from setTimeout was NodeJS.Timeout, maybe if i read it from window `window.setTimeout` we also need to fix in slick.remotemodel then, i followed the typing that was there too

Reply:
moving discussion to #14